components/style/values/animated/svg.rs

```diff
@@ -0,0 +1,92 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+//! Animated types for CSS values in SVG
+
+use app_units::Au;
+use values::animated::{Animate, Procedure};
+use values::computed::{NonNegativeNumber, Number, Percentage};
+use values::computed::length::{CalcLengthOrPercentage, LengthOrPercentage};
+use values::computed::length::NonNegativeLengthOrPercentage;
+
+/// Stroke-* value support unit less value, so servo interpolate length value as number.
```

Comment:
Why is this about `stroke-*` if the type name is about arbitrary numbers and percentages? The naming in this PR is still not good. Where exactly is this type used?

Reply:
Thank you so much!
I changed to these name to same to computed value.